be/src/storage/tablet_manager.cpp

```diff
@@ -701,6 +701,27 @@ TabletSharedPtr TabletManager::find_best_tablet_to_compaction(CompactionType com
     return best_tablet;
 }
 
+Status TabletManager::generate_pk_dump_in_error_state() {
```

Comment:
it is a little confusing reading from this function name, it says `generate_pk_dump_in_error_state`, but actually, it pulls all pk tablets without filtering their status.

And the status is filtered in `generate_pk_dump` which is actually only do the dump if tablet is error, but its name doesn't say so.

Reply:
Ok, i will change this name in next pr.